File: curso-python/ex022.py
nome = str(input("digite seu nome completo: ")).strip()

print(f"Frase em maiúsucula {nome.upper()}")
print(f"Frase em minúscula {nome.lower()}")
print(f"quantas letras ao todo?: {len(nome.replace(' ',''))}")
print(f"quantas letras tem o primeiro nome?: {len(nome.split()[0])}")

File: curso-python/javascript-exercicios/ex022.js
const prompt = require("prompt-sync")();
let nome = prompt("digite seu nome completo: ")

console.log(`frase em maiúscula ${nome.toUpperCase()}`)
console.log(`Frase em minúsculo ${nome.toLowerCase()}`)
let r = nome.replace(' ','')
console.log(`quantas letras ao todo: ${r.length}`)